### PR TITLE
create bosun guide for general meeting

### PR DESCRIPTION
**Pull request by jasonbrooks**
This is copied and adapted from the Kubernetes steering bosun guide at https://github.com/kubernetes/steering/blob/main/operations/bosun.md

@justaugustus

### CODE REVIEW

**Comment by LarryKunz** [comment]
Looks good, Jason. Two quick comments:
1. Shortly after starting the recording, we should invite new participants to introduce themselves.
2. In a couple of sections, item numbers are out of sequence.

**Comment by jasonbrooks** [comment]
Thanks, @LarryKunz I made those tweaks

File: bosun.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Inclusive Naming Initiative Biweekly Bosun Guide  
+
+This doc is a guide to help the delegated INI member bosun the public meeting.
```
Comment (quaid) [inline bosun.md:3]:
```suggestion
This doc is a guide to help the delegated INI member perform the role of /bosun/ for the public meeting. 
```

It's a noun and not a typical usage, so we'll want to define and explain following this.

File: bosun.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Inclusive Naming Initiative Biweekly Bosun Guide  
+
+This doc is a guide to help the delegated INI member bosun the public meeting. 
+INI meeting attendees will pick a new one for the next meeting at the end.
```
Comment (quaid) [inline bosun.md:4]:
```suggestion
INI meeting attendees will pick a new bosun for the next meeting at the end.
```

**Review by quaid — APPROVED** [review]
These are my language fixes for clarity, the procedure seems fine.

I would like to refactor this document to have a single sentence per line for easier diffing, but it may not matter if it's not going to change much going forward.

**Review by justaugustus — APPROVED** [review]

**Pull request merged**